src/debezium-service/values.yaml

```diff
@@ -0,0 +1,134 @@
+
+
+ingressClass130: nginx130
+namespace:
+#  name: <+infra.namespace>
+  annotated: false
+
+name: debezium-service
+envType: <+pipeline.variables.envType>
+codeVersion: <+artifact.tag>
+replicas: 1
+podMemory: 1Gi
+maxUnavailable: 0
+maxSurge: 100%
+cpu: 512m
+image: <+artifact.image>
+httpServerPort: 14552
+#pathPrefix: /<+infra.namespace>
+app_gen: ng
+ingressClass: nginx
+
+DEPLOY_MODE: KUBERNETES
+
+#domainPrefix: <+infra.namespace>-
+BASE: envSpecific
+
+STACK_DRIVER_LOGGING_ENABLED: true
+
+#redislabsCATruststore: <+secrets.getValue("qa_REDISLABS_CA_TRUSTSTORE")>
+ng_service_account: envSpecific
```

Comment:
all of the configmap data, should be inside config object.
for example , its called {{ .Values.config.EVENTS_FRAMEWORK_REDIS_SSL_ENABLED | quote }}
config: 
  EVENTS_FRAMEWORK_REDIS_SSL_ENABLED: somevalue